### PR TITLE
DevKit updates for 3.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ It's auto-generated by sonata-project/dev-kit package.
 
 # SonataDoctrineOrmAdminBundle
 
-Integrate Doctrine ORM into the SonataAdminBundle
+Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
```
Comment (phansys) [inline README.md:9]: Where this change comes from?

Comment (VincentLanglet) [reply README.md:9]: Dont know why but some changes like this happen sometimes. 
And then, they go away: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1240/files

**Review by phansys — APPROVED** [review]
I think we could merge this as is.
I have some doubts about the failures, but since we haven't any concrete clue about its origin, I guess we can assume that they are not related to these changes.

**Comment by VincentLanglet** [comment]
Maybe @franmomu could help

**Comment by phansys** [comment]
Maybe the failing test **Test / PHP 7.3 + highest + normal (pull_request)** is using a invalid cached docker layer.
I can't think about other reason for this scenario.

**Comment by jordisala1991** [comment]
Fixed re running build

**Pull request merged**